packages/browser-wallet/src/popup/popupX/pages/EarningRewards/Delegator/TransactionFlow.tsx

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable react/destructuring-assignment */
+import React, { useCallback, useState } from 'react';
+import { Location, Navigate, useLocation, useNavigate } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { AccountInfoType, DelegationTargetType } from '@concordium/web-sdk';
+
+import { absoluteRoutes } from '@popup/popupX/constants/routes';
+import MultiStepForm from '@popup/shared/MultiStepForm';
+import { useSelectedAccountInfo } from '@popup/shared/AccountInfoListenerContext/AccountInfoListenerContext';
+import { formatCcdAmount } from '@popup/popupX/shared/utils/helpers';
+import FullscreenNotice, { FullscreenNoticeProps } from '@popup/popupX/shared/FullscreenNotice';
+import Page from '@popup/popupX/shared/Page';
+import Button from '@popup/popupX/shared/Button';
+import DelegatorStake from './Stake';
+import DelegatorType from './Type';
+import { configureDelegatorPayloadFromForm, type DelegatorForm } from './util';
+import { DelegationResultLocationState } from './Result/DelegationResult';
+
+function NoChangesNotice(props: FullscreenNoticeProps) {
+    const { t } = useTranslation('x', { keyPrefix: 'earn.delegator.update.noChangesNotice' });
+    return (
+        <FullscreenNotice {...props}>
+            <Page>
+                <Page.Top heading={t('title')} />
+                {t('description')}
```

Comment:
**Suggestion:** Wrap in some text component, the font seems to be different at least.

Reply:
You're right, good catch 👍